openrndr-color/src/main/kotlin/org/openrndr/color/ColorRGBa.kt

```diff
@@ -147,6 +148,17 @@ data class ColorRGBa(val r: Double, val g: Double, val b: Double, val a: Double
      */
     val maxValue get() = r.coerceAtLeast(g).coerceAtLeast(b)
 
+    // https://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef
+    val luminance get() = 0.2126 * r + 0.7152 * g + 0.0722 * b
```

Comment:
This does not take into account in which RGB space ColorRGBa is. So it returns different luminances for the same color in sRGB and linear RGB. It looks like this is only correct for ColorRGBa in linear RGB

Reply:
Updated

Reply:
:+1: